authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/validator/AuthRequestValidator.java

```diff
@@ -76,6 +77,9 @@ public class AuthRequestValidator extends BaseAuthRequestValidator {
 	 */
 	private List<String> allowedDomainUris;
 	
+	@Value("${mosip.ida.validate-allowed-env-enabled:true}")
+	private boolean isAllowedEnvValidationEnabled;

```

Comment:
can we change the naming of this variable and property to convey the opposite and keep the default as false. 
The idea is the default mechanism should be validation enabled, and there is a separate config to disable it, so if required country will add this config and mark it as true

Reply:
Done Please check